### PR TITLE
S03 us76 secrets

### PR DESCRIPTION
**Pull request by cyrius83600**

### CODE REVIEW

**Review by AnassAmh — DISMISSED** [review]

**Review by AnassAmh — APPROVED** [review]

**Review by AnassAmh — APPROVED** [review]

File: src/app/components/header-mobile/header-mobile.component.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
               <ul class="submenu">
                 <a routerLink="home"><li>Races</li></a>
                 <a routerLink="glossaire/classes"><li>Classes</li></a>
-                <a routerLink="home"><li>Les Secrets</li></a>
+                <a routerLink="glossaire/secrets"><li>Secrets</li></a>
```
Comment (Benja-san) [inline src/app/components/header-mobile/header-mobile.component.html:48]: put a inside li tags

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
               <ul class="submenu">
                 <a routerLink="home"><li>Races</li></a>
```
Comment (Benja-san) [inline src/app/components/header/header.component.html:20]: same here

File: src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="background-image">
+  <h1 class="title">Les secrets primordiaux </h1>
+    @for(secret of secrets;track secret) {
+      <p>{{($index+1) + ".    " + secret.description}}</p>
+    }
+    <h1 class="title">Les secrets anecdotiques </h1>
```
Comment (Benja-san) [inline src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html:6]: only 1 H1 by page

File: src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<div class="background-image">
+  <h1 class="title">Les secrets primordiaux </h1>
+    @for(secret of secrets;track secret) {
```
Comment (Benja-san) [inline src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html:3]: wrap this for into ul tag and use li to wrap your paragraphs

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html (hunk 3 above)
Comment (cyrius83600) [reply src/app/pages/primordial-secrets-page/primordial-secrets-page.component.html:8]: Les 2 titres ont la meme importance d'ou les 2 H1

**Review by Benja-san — APPROVED** [review]

**Pull request merged**